### ISSUE TITLE
Fix contrast on sub-tabs

### ISSUE THREAD

**Issue by mcharg** [ONRR/onrr.gov-site#1317]
There's not enough contrast.  We should probably fix this before the demo.
![image.png](https://images.zenhubusercontent.com/5c589fe5e180620812ea22aa/b4d9904d-9916-465e-b1af-1f7f387f3036)

Changing to black or dark gray font like the wireframes should fix it.
![image.png](https://images.zenhubusercontent.com/5c589fe5e180620812ea22aa/fa650551-2a7e-4dc5-9dbf-6bcc1b517375)

**Comment by mcharg** [comment]
@jlouis-onrr Looks good.
